api/workerapi/v1/worker.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "patch/go.proto";
+
+option (go.lint).all = true;
+option (go.lint).initialisms = "ID";
+
+// Versioning via package name, for future-proofing
+package wharf.worker.v1;
+
+option go_package = "github.com/iver-wharf/wharf-cmd/api/workerapi/v1";
+
+service Worker {
+  rpc StreamLogs(LogStreamRequest) returns (stream LogStream);
+  rpc StreamStatusEvents(StatusEventRequest) returns (stream StatusEvent);
+  rpc StreamArtifactEvents(ArtifactEventRequest) returns (stream ArtifactEvent);
```

Comment:
GRPC response types should have the suffix "Response"

- LogStream -> LogStreamResponse
- StatusEvent -> StatusEventResponse
- ArtifactEvent -> ArtifactEventResponse

Reply:
Actually, they should be the RPC name + "Response" or "Request, so:



* LogStreamRequest -> StreamLogsRequest
* StatusEventRequest -> StreamStatusEventsRequest
* ArtifactEventRequest -> StreamArtifactEventsRequest
* LogStream -> StreamLogsResponse
* StatusEvent -> StreamStatusEventsResponse
* ArtifactEvent -> StreamArtifactEventsResponse